tests/integration/__tests__/cache.js

```diff
@@ -55,6 +55,19 @@ describe("--cache option", () => {
     );
   });
 
+  it("throws error when use `--cache-strategy` without `--cache`.", async () => {
+    const { stderr } = await runPrettier(
+      dir,
+      ["foo.js", "--cache-strategy", "content"],
+      {
+        input: "const a = a;",
+      }
+    );
+    expect(stripAnsi(stderr.trim())).toBe(
+      "[error] `--cache-strategy` is cannot be used without `--cache`."
```

Comment:
```suggestion
    "[error] `--cache-strategy` cannot be used without `--cache`."
```

Don't miss https://github.com/prettier/prettier/pull/12800#discussion_r889630196

Reply:
[fefe40a](https://github.com/prettier/prettier/pull/12800/commits/fefe40a7b51edbee5a09aabe2b9802d63ade40af)